recipes/_configure_indexes.rb

```diff
@@ -36,6 +36,20 @@
       :index
     end
 
+  daily_mb = hash.delete('_maxDailyDataSizeMB')
```

Comment:
Looks like we are assuming that this value should include some padding? If so should the README update say that?